src/screens/CreateWalletScreen.tsx

```diff
@@ -32,38 +32,28 @@ interface Props {
 }
 interface State {
   label: string;
-  selectedIndex: number;
+  selectedType: WalletType;
 }
 
 export class CreateWalletScreen extends React.PureComponent<Props, State> {
   state: State = {
     label: '',
-    selectedIndex: 0,
+    selectedType: '2-Key Vault',
```

Comment:
Comment to all occurrences. Imho it should be taken from wallet class type.

Reply:
I think it's better now